samples/benchmarks/coremark/sample.yaml

```diff
@@ -0,0 +1,62 @@
+sample:
+  name: CoreMark Benchmark
+  description: CoreMark sample for the CPU performance evaluation.
+               Sample runs the CoreMark benchmark that measures the CPU efficiency
+               performing different algorithms like state machine, CRC calculation,
+               matrix manipulation and list processing (find and sort).
+tests:
+  sample.benchmark.coremark:
```

Comment:
Use `common:` syntax to avoid duplications (see `samples/edge_impulse/wrapper/sample.yaml` for an example)

Reply:
discussed offline. Will clean up redundant parts after adding nrf54h20